index.html

```diff
@@ -41,19 +40,158 @@ <h1 id="idh1">
                 </a>
                 <ul class="icons">
                     <li><a href="https://twitter.com/carlosigreda" target="_blank" rel="noopener"><img
-                                src="/images/Twiter.svg" alt="Twiter"></a></li>
+                                src="./images/Twiter.svg" alt="Twiter"></a></li>
                     <li><a href="https://www.linkedin.com/in/carlos-jes%C3%BAs-igreda-rivas-323763116/" target="_blank"
-                            rel="noopener"><img src="/images/Linkedin.svg" alt="Linkedin"></a></li>
-                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="/images/Medium.svg"
+                            rel="noopener"><img src="./images/Linkedin.svg" alt="Linkedin"></a></li>
+                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="./images/Medium.svg"
                                 alt="Medium"></a>
                     </li>
                     <li><a href="https://github.com/CarlosIgreda" target="_blank" rel="noopener"><img
-                                src="/images/Github.svg" alt="Github"></a></li>
+                                src="./images/Github.svg" alt="Github"></a></li>
                     <li><a href="https://www.angellist.com/" target="_blank" rel="noopener"><img
-                                src="/images/AngelList.svg" alt="AngelList"></a></li>
+                                src="./images/AngelList.svg" alt="AngelList"></a></li>
                 </ul>
             </div>
         </section>
+        <section class="work s1">
```

Comment:
- [ ] please kindly ensure the whole work section has one parent container which holds all the projects .. inside a `section tag`. also The main container of the projects has a `display:grid`

Reply:
Done @tufoinnkuo10. I've already included all cards in a `section tag` with its respective ``display:grid``